elements/src/b-menu-button.js

```diff
@@ -20,7 +20,7 @@ export default class MenuButtonElement extends HTMLElement {
   }
 
   get button() {
-    return this.querySelector('button')
+    return this.querySelector('b-listbox button')
```

Comment:
This won't fix the issue: this selector would select the first button in the menu, but the toggle menu should be outside of the menu. I fixed the issue a few minutes ago btw, thanks for taking the time for this PR though!